src/guides/other/commonErrors.md

```diff
@@ -0,0 +1,499 @@
+# Common BDFD Errors
+
+---
+
+# Error Messages Handling 
+
+## Error message syntax
+
+### Error function
+
+The function that is causing the error.
+
+**Function location:**
+![Location of function](https://media.discordapp.net/attachments/1021982737424449586/1255928553921576992/syntax_msg_error-function.png?ex=667eea1d&is=667d989d&hm=295240ddc00f1c0a97f2a87940bab47d691b958b21c6b23ede29f61da45542d4&)
+
+
+### Row and Column
+
+- `Row`: 
+The row where the error is located.
+- `Column`:
+The column where the error is located.
+
+**Row and column location:**
+![Location of row and column](https://media.discordapp.net/attachments/1021982737424449586/1255955254789537852/syntax_msg_error-rowcolumn.png?ex=667f02fb&is=667db17b&hm=4ca5a29b2576e824e65db688973e662549f5e721b157b88f8cee4920b529bb97&)
+
+
+*In this case, the row is '3' and the column '15'.*
+
+## Argument types and flags
+
+### Argument types
+
+The argument type represent different values, which indicates what the argument value must be.
+
+> You can check the argument type list [here](https://nilpointer-software.github.io/bdfd-wiki/guides/other/arguments.html#argument-types).
+
+### Argument flags
+
+The argument flags mark if arguments within a function may be used (or hence, not used at all).
+
+> You can check the argument flags list [here](https://nilpointer-software.github.io/bdfd-wiki/guides/other/arguments.html#argument-flags).
+
+---
+
+# Expectations
+
+## Aclaration
```

Comment:
What is "Aclaration"?

Reply:
Is a way of saying "Making it clear" or "Clarifying".